### ISSUE TITLE
build failure

### ISSUE THREAD

**Issue by yuwata** [systemd/systemd#26757]
### systemd version the issue has been seen with

HEAD

### CPU architectures issue was seen on

None

### Unexpected behaviour you saw

https://github.com/systemd/systemd/actions/runs/4391507839/jobs/7690708783
```
ninja: Entering directory `systemd/build'
ninja: error: unknown target 'bootctl', did you mean 'oomctl'?
Error: Process completed with exit code 1.
```

**Comment by medhefgo** [comment]
@DaanDeMeyer The mkosi ci action needs python-pyelftools.

**Comment by medhefgo** [comment]
Alternatively, we could also split the `bootloader` meson option into two: one for userspace stuff and the other for actual efi binaries.

**Comment by DaanDeMeyer** [comment]
https://github.com/systemd/mkosi/pull/1382

**Comment by DaanDeMeyer** [comment]
@medhefgo Slightly unrelated, but I don't know how to contact you anywhere else. Looking at elf2efi.py, You seem to be doing all the PE setup without any libraries or objcopy involved. Any chance you could modify ukify to use the same approach so we can get rid of the dependencies on binutils and python-pefile if that's not too complicated?

**Comment by DaanDeMeyer** [comment]
(Or alternatively use python-pefile in elf2efi I guess to keep things consistent)

**Comment by medhefgo** [comment]
You can always reach me by mail (it's in git logs).

Sadly, python-pefile has very limited writing abilities (afaik it can only overwrite header entries). We need to create an image from scratch, though.

I was thinking of doing that as it would also make ukify work for every arch.

**Comment by yuwata** [comment]
Fixed with [systemd/mkosi#1382](https://github.com/systemd/mkosi/pull/1382) and #26758.
